ios/Plugin/Plugin.swift

```diff
@@ -126,6 +126,52 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         }
     }
 
+    @objc func getBeaconsPermissionStatus(_ call: CAPPluginCall) {
+        getLocationPermissionsStatus(call)
+    }
+
+    @objc func requestBeaconPermissions(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            self.locationManager.requestWhenInUseAuthorization()
+            call.resolve()
+        }
+    }
+
+    @objc override public func checkPermissions(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            let authorizationStatus = CLLocationManager.authorizationStatus()
+            var location = "prompt"
```

Comment:
what are these prompt strings used for?

Reply:
These are referenced in the documentation for this method in capacitor. https://capacitorjs.com/docs/plugins/ios#checkpermissions